src/Microsoft.AspNet.OData.Shared/ODataAPIHandler.cs

```diff
@@ -0,0 +1,186 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="ODataAPIHandler.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.OData.Edm;
+using Microsoft.OData.UriParser;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// The handler class for handling users' create, delete and update methods.
+    /// This is the handler for data modification where there is a CLR type.
+    /// </summary>
+    public abstract class ODataAPIHandler<TStructuralType>: IODataAPIHandler where TStructuralType : class
+    {
+        /// <summary>
+        /// TryCreate method to create a new object.
```

Comment:
This being public documentation for this method, describe briefly what the method does. I don't think using `TryCreate` in the documentation for a method named `TryCreate` is the way to do it